cloud-common/src/main/resources/application.yml

```diff
@@ -120,18 +120,10 @@ cb:
 
   blueprint:
      defaults: >
-                EDW-ETL: Apache Hive, Apache Spark 2=hdp26-etl-edw-spark2;
-                EDW-Analytics: Apache Hive 2 LLAP, Apache Zeppelin=hdp26-edw-analytics;
-                Data Science: Apache Spark 2, Apache Zeppelin=hdp26-data-science-spark2;
```

Comment:
Silly question: What happens when "HDP 2.6" is chosen in the "Platform Version" dropdown (step "General Configuration" of Cluster Create)? Should not that option be removed as well?